test/mongodb-fn-execute.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs');
```

Comment:
Is there any reason why we would include the test in `modules` so it would be applied to all adapters?

Reply:
modules get loaded all at once - and it has to process this test differently because it has to make a new instance of the adapter.  so this now will run against all adapters, but it isn't in modules.